elasticsearch_dsl/search_base.py

```diff
@@ -363,13 +363,22 @@ def __getitem__(self, n):
         # Elasticsearch won't get all results so we default to size: 10 if
         # stop not given.
```

Comment:
This comment can be removed now as it has been duplicated in the `slice_start is not None` case